containers/support/AttachScreenshot.js

```diff
@@ -0,0 +1,63 @@
+import React, { useState } from 'react';
+import { c } from 'ttag';
+import { Text, Icon, Button, FileInput, useNotifications } from 'react-components';
+import { toBlob, resize } from 'proton-shared/lib/helpers/image';
+import { MAX_SIZE_SCREENSHOT } from 'proton-shared/lib/constants';
+import PropTypes from 'prop-types';
+
+const AttachScreenshot = ({ id, onUpload, onReset }) => {
+    const [attached, setAttached] = useState(false);
+    const { createNotification } = useNotifications();
+
+    const handleClick = () => {
+        setAttached(false);
+        onReset();
+    };
+
+    const handleChange = async ({ target }) => {
+        const images = [...target.files].filter(({ type }) => /^image\//i.test(type));
+
+        if (images.length) {
+            setAttached(true);
+            onUpload(
+                await Promise.all(
+                    images.map((img) =>
+                        resize(img, MAX_SIZE_SCREENSHOT).then((base64str) => ({
+                            name: img.name,
+                            blob: toBlob(base64str)
+                        }))
+                    )
+                )
+            );
+        } else {
+            createNotification({
+                type: 'error',
+                text: c('Error notification in the bug report modal when the user upload file').t`No image selected`
+            });
+        }
+    };
+
+    if (attached) {
+        return (
+            <>
+                <Text>
+                    <Icon name="insert-image" /> {c('Info').t`Screenshot(s) attached`}
```

Comment:
Maybe we can display a small preview (~ 500*400) ?

Reply:
I let design team design.